lib/cassieq/client/messages.rb

```diff
@@ -3,16 +3,18 @@
 module Cassieq
   class Client
     module Messages
-      def publish_message(queue_name, message)
-        request(:post, "queues/#{queue_name}/messages", message)
+      def publish_message(queue_name, message, initial_invis_time = nil)
+        query = { initialInvisibilityTime: initial_invis_time } unless initial_invis_time.nil?
```

Comment:
We want this to be initialize invisiblity seconds, the `time` suffix is deprecated